### DISCUSSION TITLE
Broken web interface in v5.7.18

### DISCUSSION THREAD

**Discussion by jacopo-j** [firefly-iii/firefly-iii#6832] [Firefly III]
Hi, I've started experiencing an issue some time after updating to version 5.7.18. I'm not sure whether this is a bug with Firefly or an issue with my setup (which would be weird, since I didn't change anything). 

The issue is that Firefly does not load resources related to the HTML pages (stylesheets, fonts, etc.) when visiting pages other than the home page.

![Screenshot](https://i.imgur.com/7v6agpv.png)

Notice that the path from which Firefly attempts to load resources is wrong. It looks like Firefly attempts to load resources relative to the current path. In other words, it requests URL `https://<host><current_path>/v1/lib/bs/css/...` but it should be `https://<host>/v1/lib/bs/css/...`.

Any idea?

**Comment by JC5** [comment]
Could you share the output from the /debug page?

**Comment by jacopo-j** [comment]
Debug information generated at 2023-01-11 10:18:06 Europe/Amsterdam for Firefly III version **5.7.18**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.7.18 |
| Firefly III API | 1.5.6 |
| Build | 346, 02-01-2023 16:12:50 CET |
| DB version | 18 (exp. 18) |
| PHP | `8.1.13` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Amsterdam |
| Browser TZ | Europe/Rome |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | debug, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start |  |
| Session end |  |
| Session first |  |
| User ID | 1 |
| User language | it_IT |
| User locale | it_IT |
| Attempt at "it_IT.utf8" | 'it_IT.utf8' |
| Attempt at "it_IT.UTF-8" | 'it_IT.UTF-8' |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15 |

**Comment by JC5** [comment]
Mmmm, not really sure what the issue could be. In the source of the dashboard, what's the content of <base href"">? And what is it on a sub-page?

**Comment by jacopo-j** [comment]
It is `http://<host>/` in both cases. Note that it is `http` although the server is configured to use HTTPS only (it has HTTP to HTTPS redirects). I don't know if that could be relevant.

**Reply by JC5** [reply]
In that case, your reverse proxy is configured incorrectly. 

https://docs.firefly-iii.org/firefly-iii/installation/docker/#docker-and-reverse-proxies
